File: code/94104.pas
program Bai104;
var a, b: real;
x: real;
begin
  readln(a,b);
  if a <> 0 then writeln('Nghiem duy nhat x =   ',-b/a:5:3 )
    else if b <> 0 then writeln('Phuong trinh vo nghiem') else 
  writeln ('Nghiem voi moi x');
end.

File: code/94104.py
a, b= [int(x) for x in input().split()]
if (a != 0) and  (b!= 0):
  print("%.3f" % (-b/a))
else:
  if (a == 0) and (b != 0):
    print('VO NGHIEM')
  else:
     print('NGHIEM VOI MOI x')